README.md

```diff
@@ -381,7 +383,8 @@ const factory = Model.UserFactory.getInstance({
 });
 const config = {
     params: {
-        username: 'batman'
+        username: 'batman',
+        scmContext: 'github:github.com'
```

Comment:
What is the significance of the `github` before the colon vs the `github.com` after it?